sdk/template/azure-template/inc/azure/template/version.hpp

```diff
@@ -8,7 +8,7 @@
 #define AZURE_TEMPLATE_VERSION_MAJOR 1
 #define AZURE_TEMPLATE_VERSION_MINOR 0
 #define AZURE_TEMPLATE_VERSION_PATCH 0
-#define AZURE_TEMPLATE_VERSION_PRERELEASE "beta.8"
+#define AZURE_TEMPLATE_VERSION_PRERELEASE "beta.22"
```

Comment:
high jump :)

Reply:
Even if you set up parts of the pipeline on your local machine to replicate the real pipelines it's hard to be certain that your local setup is accurately replicating the production behavior.